backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/sort/impl/LuceneTextFieldSortBuilder.java

```diff
@@ -36,7 +37,14 @@ protected Object encodeMissingAs(F converted) {
 	public void buildAndContribute(LuceneSearchSortCollector collector) {
 		// For STRING type, missing value must be either STRING_FIRST or STRING_LAST.
 		// Otherwise we need a CUSTOM type.
-		collector.collectSortField( ( useMissingValue() ) ? customType() : stringType(), nestedDocumentPath );
+		collector.collectSortField( getSortField(), nestedDocumentPath );
+	}
+
+	private SortField getSortField() {
```

Comment:
Maybe rather `createSortField()`? It's always confusing to have a getter that doesn't just "get". Also, you're going to add side-effects to this getter in later commits.